pkg/agent/manifest_test.go

```diff
@@ -0,0 +1,23 @@
+package agent
+
+import "testing"
+
+func TestImageResolve(t *testing.T) {
+	tests := []struct {
+		systemDefaultRegistry string
+		image                 string
```

Comment:
I think it would be easier to read if the struct follows the succession of the `resolve` function parameters. So the second should be `privateRepoURL` and the third `image`.

Reply:
Yes, I was just too lazy to re-order the strings :)